docs/conf.py

```diff
@@ -49,6 +49,13 @@
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 
+# Add note about documentation from development branch
+if os.environ.get("READTHEDOCS_VERSION") == "development":
+    rst_prolog = """.. attention::
+
+        This page was created from `development <https://github.com/software-mansion/starknet.py>`_ branch.
+    """
+
```

Comment:
Looks okay, but was it tested at all?

Reply:
It has been tested in my underground laboratory 🧪

Reply:
Could you show us a screenshot? 😏

Reply:
https://starknetpy.readthedocs.io/en/development/